CODEOWNERS

```diff
@@ -74,7 +72,7 @@
 /.bazelrc @magma/approvers-bazel
 *.bazel @magma/approvers-bazel
 
-/CODEOWNERS @amarpad @electronjoe @hcgatewood
+/CODEOWNERS @magma/repo-magma-admin
```

Comment:
Which one would be the appropriate group to be placed here?
(I only put a suggestion here to be able to write this comment and start a discussion.)

I believe @amarpad has a new project since November 2021 as there were only 5 PR reviews since than.
@hcgatewood @electronjoe Would you be able to suggest a appropriate group?

Reply:
maybe a tsc group?

Reply:
Changed to `@magma/approvers-tsc` in expectation of the group getting set up by @wendallkaters .